test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -649,6 +649,7 @@ Function sys.translate_pg_type_to_tsql(oid)
 Function sys.trigger_nestlevel()
 Function sys.trigger_test_fault()
 Function sys.tsql_get_constraintdef(oid)
+Function sys.tsql_get_expr(text,oid)
```

Comment:
it means this object does not have dependency tests in upgrade test path

Reply:
This is not customer facing function , so we don’t actually need dependency tests on it.